cmd/wnode-status/config.go

```diff
@@ -16,98 +45,66 @@ func makeNodeConfig() (*params.NodeConfig, error) {
 		return nil, err
 	}
 
-	// TODO(divan): move this logic into params package?
-	if *nodeKeyFile != "" {
-		nodeConfig.NodeKeyFile = *nodeKeyFile
-	}
+	nodeConfig.ListenAddr = *listenAddr
 
-	// disable log
-	nodeConfig.LogLevel = "CRIT"
-	nodeConfig.LogFile = ""
+	nodeConfig.LogLevel = *logLevel
+	if filepath.IsAbs(*logFile) {
+		nodeConfig.LogFile = *logFile
+	} else if *logFile != "" {
+		nodeConfig.LogFile = filepath.Join(*dataDir, *logFile)
+	}
 
 	// disable les and swarm for wnode
 	nodeConfig.LightEthConfig.Enabled = false
 	nodeConfig.SwarmConfig.Enabled = false
 
-	nodeConfig.RPCEnabled = *httpEnabled
-
 	// whisper configuration
 	whisperConfig := nodeConfig.WhisperConfig
-
 	whisperConfig.Enabled = true
 	whisperConfig.IdentityFile = *identity
-	whisperConfig.PasswordFile = *password
-	whisperConfig.EchoMode = *echo
-	whisperConfig.BootstrapNode = *bootstrap
-	whisperConfig.ForwarderNode = *forward
```

Comment:
@adambabik I've rechecked this code - you defenetly right in removing it. Could you create anre issue 'add forvarder and bootstrap modes' to offline mailboxing?

Reply:
@JekaMas but do we need these modes for anything? I can't figure out any use case. Maybe except for bootstrap node, when it would not discover and connect to any other nodes by itself.

Reply:
Bootstrap is useful.

Reply:
@JekaMas more details, please. Describe at least one use case it's useful for Offline Messages MVP. I created an issue https://github.com/status-im/status-go/issues/486 but I only figure out "testing" use case.